mlflow/tracking/fluent.py

```diff
@@ -103,6 +103,7 @@ def start_run(run_uuid=None, experiment_id=None, source_name=None, source_versio
              the run's state.
     """
     global _active_run_stack
+    experiment_id = str(experiment_id) if experiment_id is not None else None
```

Comment:
This doesn't seem quite right. Maybe we should just put `str()` around `experiment_id` on L124?